src/pages/workspace/WorkspaceMembersPage.js

```diff
@@ -116,12 +139,20 @@ function WorkspaceMembersPage(props) {
         if (removeMembersConfirmModalVisible && !_.isEqual(accountIDs, prevAccountIDs)) {
             setRemoveMembersConfirmModalVisible(false);
         }
-        setSelectedEmployees((prevSelected) =>
-            _.intersection(
-                prevSelected,
+        setSelectedEmployees((prevSelected) => {
+            // Filter all personal details in order to use the elements needed for the current workspace
+            const currentPersonalDetails = filterPersonalDetails(props.policyMembers, props.personalDetails);
+            // Checking selected elements. Necessary in cases where there is a transition from offline to online mode, since after this the id changes for new members
```

Comment:
I think this comment could be better: "We need to filter the previous selected employees by the new personal details, since unknown/new user id's change when transitioning from offline to online